imap_processing/codice/l0/__init__.py

```diff
@@ -0,0 +1,515 @@
+LOSSY_A_TABLE = {
```

Comment:
Where did you get these numbers from? It might be nice to put a reference to the specific page number of the algorithm document / sheet for easy reference in the future.

Reply:
Good idea, I will add a note. Unfortunately these are not defined in any sort of official document -- They were given to us via email from Greg Dunn.